dvc/command/imp.py

```diff
@@ -17,6 +17,7 @@ def run(self):
 
             self.project.imp(self.args.url, out)
         except DvcException:
-            logger.error('failed to import {}'.format(self.args.url))
+            logger.error('failed to import {}. You can add it with `dvc add` '
```

Comment:
It seems a little misleading. If import failed, we should probably tell the user that he can download it himself and `dvc add` it as an alternative.

Reply:
right